autoload/ale/handlers/textlint.vim

```diff
@@ -0,0 +1,18 @@
+" Author: tokida https://rouger.info
+" Description: Redpen, a proofreading tool (http://redpen.cc)
+
+function! ale#handlers#textlint#HandleTextlintOutput(buffer, lines) abort
+    let l:res = json_decode(join(a:lines))[0]
```

Comment:
Move this callback into the `ale_linters` file, like other linters. Use `ale#util#FuzzyJSONDecode(a:lines, [])` to get the list of results and then `get(l:whatever, 0, {'messages': []})` on that. Add a test with some empty output.